code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -358,6 +358,9 @@
 	var/lower_fuel_limit = 800
 	var/upper_fuel_limit = 1000
 
+/obj/item/explosive/grenade/flare/dissolvability(acid_strength)
```

Comment:
I dislike that being a proc, why isn't that just a var

Reply:
@BraveMole Tivi normally reees at atom level vars. Also this is used for more than just a number.

Reply:
Ok but anyway, sometimes this procs takes an acid_strenght, sometimes it takes an acid_type so either way it has some issues